doi-transformer/src/test/java/no/unit/nva/doi/transformer/model/crossrefmodel/CrossrefDocumentTest.java

```diff
@@ -47,5 +48,19 @@ public void testSettersAndGettersCrossrefBook() throws IOException {
 
     }
 
+    @Test
+    public void testSettersAndGettersCrossrefBookChapter() throws IOException {
```

Comment:
If it only to test setters and getters for mere test coverage, one could use `@JacocoGenerated` annoation. Testing JsonSerialiazation is another thing though.